lib/Graph/Graph.cpp

```diff
@@ -575,16 +575,15 @@ Function::createRowwiseQuantizedFullyConnected(Context &ctx,
   // it is assumed to be quantized data and the scale and offset should be
   // provided. But for rowwise quantization, the scales and offsets are stored
   // in vectors separately, we add the dummy scale and offset here.
-  auto *qWeights = getParent()->createVariable(ElemKind::Int8QTy, W->dims(),
-                                               0.0, 0, "weights.rwqfc",
-                                               VisibilityKind::Private, false);
+  auto *qWeights = getParent()->createPlaceholder(
+      ElemKind::Int8QTy, W->dims(), 0.0, 0, "weights.rwqfc", false);
```

Comment:
Here qWeights is a constant. That is why I used "createVariable" for it.

Reply:
Hm, I see your point.  Would this member ever be trained?  (i.e., could it be non-constant in the future?)

Reply:
I don't think that we can train quantized operators.

Reply:
No. The row-wise quantization is only for const weights so far.